packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/site-resolver.ts

```diff
@@ -1,4 +1,4 @@
-import { SiteResolver, SiteInfo } from '@sitecore-jss/sitecore-jss-nextjs';
+import { SiteResolver, SiteInfo } from '@sitecore-jss/sitecore-jss-nextjs/middleware';
```

Comment:
Had to import from `/middleware`, cause client-side stuff should be excluded from middleware execution

Reply:
For me it looks fine, in any case, it's related to middlewares, so should be ok